Competition2017/src/competition/subsystems/drive/commands/StopDriveCommand.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.drive.commands;
+
+import competition.subsystems.drive.DriveSubsystem;
+
+public class StopDriveCommand extends BaseDriveCommand {
+
+    public StopDriveCommand(DriveSubsystem driveSubsystem) {
```

Comment:
Ooh, final catch: this needs `@Inject`. Could you add a basic test which injects this command and confirms that it stops the motors? This should work, with some autocomplete mixed in where I can't remember method names:

```java
DriveSubsystem drive = // inject this
StopDriveCommand stopCommand = // inject this

drive.tankDrivePowerMode(0.5, 0.7);

// Assert that the powers were set -- there's probably a base test class for the drive which makes this easy

stopCommand.initialize();
assertTrue(stopCommand.isFinished());

// Assert that powers are zero
```